src/Compilers/CSharp/Portable/Symbols/Metadata/PE/SymbolFactory.cs

```diff
@@ -69,6 +69,10 @@ internal override TypeSymbol GetSZArrayTypeSymbol(PEModuleSymbol moduleSymbol, T
                 return elementType;
             }
 
+            // PROTOTYPE(NullableReferenceTypes):
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): [](start = 12, length = 37)

I believe this comment is misplaced. This component should not deal with nullable annotations

Reply:
I'll stop by to discuss Monday.

---
In reply to: [183127832](https://github.com/dotnet/roslyn/pull/25928#discussion_r183127832) [](ancestors = 183127832)